cloud/cmd/cloudcore/app/version.go

```diff
@@ -0,0 +1,93 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+	"k8s.io/klog"
+
+	"github.com/kubeedge/kubeedge/pkg/version"
+)
+
+const (
+	// DefaultErrorExitCode defines exit the code for failed action generally
+	DefaultErrorExitCode = 1
+)
+
+func NewCmdVersion() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Print the version of kubeadm",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := RunVersion(os.Stdout, cmd)
+			CheckErr(err, fatal)
+		},
+	}
+	cmd.Flags().StringP("output", "o", "", "Output format; available options are 'yaml', 'json' and 'short'")
+	return cmd
+}
+
+// RunVersion provides the version information of kubeadm in format depending on arguments
+// specified in cobra.Command.
+func RunVersion(out io.Writer, cmd *cobra.Command) error {
+	v := version.Get()
+
+	const flag = "output"
+	of, err := cmd.Flags().GetString(flag)
+	if err != nil {
+		klog.Fatalf("error accessing flag %s for command %s: %v", flag, cmd.Name(), err)
```

Comment:
(non blocking)
I think it would be better to replace `Fatalf` with `Errorf` and `return err`; I think this could increase the readability. Thoughts?

Reply:
@subpathdev  as @kevin-wangzefeng  said, we need to make flag `--version` install of command `version`, so this file has been deleted